app/assets/javascripts/topology.js

```diff
@@ -0,0 +1,79 @@
+var React = require('react');
+
+var Topology = React.createClass({
+  DEFAULT_DELTA: '...',
+
+  getInitialState: function() {
+    return { delta: this.DEFAULT_DELTA };
+  },
+  processDeltas: function(deltas) {
+    console.log('parse deltas', deltas);
+    var healthyPartitions = 0,
+        unhealthyPartitions = 0,
+        totalDelta = 0;
+
+
+    deltas.forEach(function(delta) {
+      var storm = delta.storm;
+      if ( storm === null ) {
```

Comment:
Another point to discuss, should it be a percentage thing?

Right now I defined healthy as "storm === null", which means it isn't processing tuples at all. Just something to think about in the future, we don't expose the healthy/unhealthy (yet)